react/features/conference/components/web/Subject.js

```diff
@@ -71,8 +171,11 @@ class Subject extends Component<Props> {
  */
 function _mapStateToProps(state) {
     const participantCount = getParticipantCount(state);
+    const domainSpeaker = getDominantSpeaker(state);
 
     return {
+        id: domainSpeaker && domainSpeaker.id,
```

Comment:
it can be refafct with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) and can be marked as local.

```suggestion
        _id: domainSpeaker?.id ?? '',
```

Also nullish coalescing is according to the line 42:
https://github.com/aeternity/jitsi-meet/blob/d4a0fae1e5124f0e609c7ce499ba54fbf4c719a2/react/features/conference/components/web/Subject.js#L39-L42